include/tvm/runtime/profiling.h

```diff
@@ -573,6 +573,8 @@ PackedFunc ProfileFunction(Module mod, std::string func_name, int device_type, i
  *        minimum duration requirement of one `repeat`.
  *        i.e., When the run time of one `repeat` falls below this time,
  *        the `number` parameter will be automatically increased.
+ * \param max_repeat_ms The maximum number of repeats when measured time is equal to 0.
```

Comment:
Let's change the word "repeat" in "max_repeat_num" or change the name completely. It looks similar but does not apply to the other arguments(repeat, min_repeat_ms, repeats_to_cooldown). It may confuse users.

Reply:
Done